source/interfaces/Server.ts

```diff
@@ -1,6 +1,6 @@
-import Route from './Route';
+import RouteProperties from './RouteProperties';
 
 export default interface Server {
-  routes: (routes: Route[]) => void;
+  routes: (routes: RouteProperties[]) => void;
```

Comment:
This will cause BREAKING CHANGES, right?

Because in my project I will have to do this update:
```diff
- import { MethodType, Route } from 'the-fake-backend';
+ import { MethodType, RouteProperties } from 'the-fake-backend';

- const routes: Route[] = [
+ const routes: RouteProperties[] = [
  {
    path: '/user',
    methods: [
      {
        type: MethodType.GET,
      },
    ],
  },
];
```

Reply:
Kind of, both interfaces are compatible.